edgePy/io/_io.py

```diff
@@ -7,12 +7,55 @@
 
 
 __all__ = [
-    'Importer',
-    'get_dataset_path'
+    'GroupImporter',
+    'DataImporter',
+    'get_dataset_path',
+    'get_open_function'
 ]
 
 
-class Importer(object):
+def get_open_function(filename):
```

Comment:
Docstring on use?

Reply:
You'll have to help me figure that out.  Not sure what's needed here.